backends-velox/src/test/scala/org/apache/gluten/execution/ScalarFunctionsValidateSuite.scala

```diff
@@ -123,6 +123,54 @@ abstract class ScalarFunctionsValidateSuite extends FunctionsValidateSuite {
     }
   }
 
+  testWithSpecifiedSparkVersion("Test array_append function - INT", Some("3.4")) {
+    withTempPath {
+      path =>
+        Seq[(Array[Int], Int)](
+          (Array(2, 1), 0),
+          (Array(1), 1),
+          (Array(), 0),
+          (Array(1, 2, null.asInstanceOf[Int]), 1),
+          (Array(null.asInstanceOf[Int]), 1),
+          (Array(null.asInstanceOf[Int]), null.asInstanceOf[Int]),
+          (Array(), null.asInstanceOf[Int])
```

Comment:
please add `(null, 1)` to test, as vanilla spark return null for `SELECT array_append(CAST(null as Array<Int>), 2);`

Reply:
ok, added